modules/lib/lib-portal/src/main/resources/lib/xp/portal.ts

```diff
@@ -487,21 +431,14 @@ interface GetCurrentContentHandler {
  *
  * @returns {object|null} The current content as JSON.
  */
-export function getContent<Data = Record<string, unknown>, Type extends string = string>(): Content<Data, Type> | null {
+export function getContent<
+    Data = Record<string, unknown>,
+    Type extends string = string,
+    Config extends object = object,
+    Regions extends Record<string, Region> = Record<string, Region>
```

Comment:
Yes! This is the best way to implement this for simple usage by the app developer. It's because when creating a controller for a page (not a part etc), the common thing to do is to grab the config from `content.page` that we got from `portalLib.getContent()`.